src/main/java/org/broadinstitute/hellbender/tools/walkers/filters/VariantFiltration.java

```diff
@@ -206,6 +207,13 @@ private void initializeVcfWriter() {
         final Set<VCFHeaderLine> hInfo = new LinkedHashSet<>();
         hInfo.addAll(getHeaderForVariants().getMetaDataInInputOrder());
 
+        // need AC, AN and AF since output if set filtered genotypes to no-call
+        if ( setFilteredGenotypesToNocall ) {
```

Comment:
This code is identical to some code below it, could it be factored into a method in GATKVariantContextUtils?  Or possible added to a new setFilteredGenotypeToNoCallArgumentCollection.